content/_index.md

```diff
@@ -2,65 +2,41 @@
-GitBOM is neither [git](/glossary/git) nor an [SBOM](/glossary/sbom).
+**GitBOM** is creative re-purposing of [git](https://en.wikipedia.org/wiki/Git)'s [directed acyclic graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph) to enable **automatic verifiable artifact resolution** across today's language-heterogeneous open source supply chains.
 
-GitBOM is a minimalistic scheme for [build tools](/glossary/build_tool) to:
+{{% notification type="info" %}}
+**Note that GitBOM is not a replacement for [SBOM](/glossary/sbom)s**, nor is it a version control system or a signing scheme.
 
-1. Build a compact [artifact tree](/glossary/artifact_tree), tracking every source code file incorporated into each built [artifact](/glossary/artifact).
-2. Embed an id for that artifact tree into the artifact being built.
-3. In a language-heterogenous environment, across packaging formats, with zero developer effort, involvement, or awareness.
+It is compatible with, and augments, these classes of tools.
+{{% /notification %}}
 
-- GitBOM is not an SBOM. It is designed to complement SBOMs, such as [SPDX](https://spdx.dev/), [CycloneDX](https://cyclonedx.org/), or [SWID](https://nvd.nist.gov/products/swid).
-- GitBOM is not a version control system. It is designed to co-exist with, but not require, them.
-
-GitBOM applies the [Unix Philosophy](https://en.wikipedia.org/wiki/Unix_philosophy) of "do one thing, and do it well." GitBOM constructs [artifact trees](/glossary/artifact_tree).
+Contrary to the name's appearance, GitBOM is neither [git](/glossary/git) nor an [SBOM](/glossary/sbom) -- it is a minimalistic scheme for all [build tools](/glossary/build_tool) to:
 
+1. Generate a compact [artifact tree](/glossary/artifact_tree) which correlates source and object files involved in the generation of any built [artifact](/glossary/artifact).
+2. Embed a unique content-addressable reference for that [artifact tree](/glossary/artifact_tree) into the [artifact](/glossary/artifact) at build time.
+3. And do this consistently across languages, environments, and packaging formats, with zero developer effort, involvement, or awareness.
 {{< /home/section >}}
 
 {{< home/section title="Why?" >}}
-By correlating every piece of software to a verifiable, complete, and concise artifact tree, GitBOM enables: 
-
-- Run-time detection of potential vulnerabilities in executables/containers by scanning a "fingerprint" rather than the contents.
-- Identifying license obligations across both open source and commercial software
-- More reliable attestation
-- Post exploit forensics
+ GitBOM will enable detailed o be able to easily, quickly, and at run time, know what components answer questions like "Does this product contain `log4j`?"Nearly every open source project has a complex list of dependencies, often maintained by different organizations and written in different languages. This is known as the **software supply chain**. We like to think of it as a fractal forest.
```

Comment:
What is the 'o' in 'GitBOM will enable detailed o be'

Reply:
huh, snippet on the editing room floor. lemme fix.